hw18.py

```diff
@@ -0,0 +1,121 @@
+class Bot:
+    def __init__(self, name):
+        self.name = name
+
+    def send_message(self, message):
+        print(message)
+
+    def say_name(self):
+        print(self.name)
+
+
+class TelegramBot(Bot):
+    url = chat_id = None
```

Comment:
nit: better to add them in the constructor and make them available to set while initiating the class
```suggestion
    def __init__(self, name, url=None, chat_id=None):
        ...
```